Update selecting.md with TH Lübecks (University of Applied Sciences Lübeck) image collection (#2110)
jupyter/docker-stacks: docs/using/selecting.md

### Other GPU-accelerated notebooks

| Flavor                       | Description                                                                                                                                                                                                                                                                                                                                                  |
| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| [GPU-Jupyter][gpu]           | Power of your NVIDIA GPU and GPU calculations using Tensorflow and Pytorch in collaborative notebooks. This is done by generating a Dockerfile that consists of the **nvidia/cuda** base image, the well-maintained **docker-stacks** that is integrated as a submodule, and GPU-able libraries like **Tensorflow**, **Keras** and **PyTorch** on top of it. |

> | [JupyterHub Images][gpu_thl] | A more fine grained collection of images based on the **docker-stacks**. They are built and maintained at the [myLab TH Lübeck](https://mylab.th-luebeck.de) using build scripts similar to iot-salzburg. Also includes several images with gpu libraries included                                                                                           |

Reviewer: ```suggestion
| [myLab TH Lübeck JupyterHub Images][gpu_thl] | Collection of images based on the **jupyter/docker-stacks**, built and maintained at the [myLab TH Lübeck](https://mylab.th-luebeck.de) using build scripts similar to iot-salzburg. Also includes several images with gpu libraries                                                                                           |
```

Reviewer: I slightly updated the description to make it shorter.
